Makefile

```diff
@@ -25,127 +27,130 @@ export GO111MODULE=on
 # Default: use GOPROXY to speed up downloading go mod dependency.
 export GOPROXY=https://goproxy.io
 
-clean:
+help:  ## Display this help information
```

Comment:
I am curious this section`/^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) }`. I think it's useless. 

Maybe you want add `##@` for grouping comments? IMOP, it's useless for now.

Reply:
Yes, it is for future grouping.
If you think it doesn't make sense, I can take it out for now. WDYT?

Reply:
And then could you please add a comment?

Reply:
How about "# NB. "##@" will be used to group make goals in the future." ?

Reply:
Or just delete `/^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) }` ?

Reply:
In fact, I prefer to remove it. And we can add it if we need in the future. WDYT?

Reply:
Already done :)